src/Hyperion/Extensions/TypeEx.cs

```diff
@@ -69,8 +94,15 @@ public static bool IsHyperionPrimitive(this Type type)
         }
 
 #if NETSTANDARD16
-    //HACK: the GetUnitializedObject actually exists in .NET Core, its just not public
-        private static readonly Func<Type, object> getUninitializedObjectDelegate = (Func<Type, object>)
+        //HACK: IsValueType does not exist for netstandard1.6
+        private static bool IsValueType(this Type type)
+            => type.IsSubclassOf(typeof(ValueType));
+        
+        private static bool IsSubclassOf(this Type p, Type c)
```

Comment:
Add missing type functions to netstandard 1.6, needed for the type blacklisting feature.